arch/arm/src/nrf52/.gitignore

```diff
@@ -0,0 +1,4 @@
+CMSIS_5*
```

Comment:
We can remove the libraries you are no longer fetching here.

Reply:
Right, thanks